### ISSUE TITLE
Security Hub Standards are not applied at the OU level

### ISSUE THREAD

**Issue by jaristizabalc** [awslabs/landing-zone-accelerator-on-aws#432]
**Describe the bug**
Using the following configuration on security-config.yaml

securityHub:
    enable: true
    regionAggregation: true
    excludeRegions: []
    standards:
      - name: AWS Foundational Security Best Practices v1.0.0
        # https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-standards-fsbp-controls.html
        deploymentTargets:
         organizationalUnits:
           - Staging
        enable: true
      - name: CIS AWS Foundations Benchmark v1.4.0
        # https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html
        deploymentTargets:
         organizationalUnits:
           - Staging        
        enable: true
      - name: NIST Special Publication 800-53 Revision 5
        # https://docs.aws.amazon.com/securityhub/latest/userguide/nist-standard.html        
        enable: true

The intent is to apply Security Standards to a specific OU for onboarding brownfield accounts.

**To Reproduce**
Release change to apply settings.

**Expected behavior**
Standards are not enabled, only the AWS Foundational Security Best Practices v1.0.0, and it is applied at the root level. It looks like the deploymentTargets option does not have any effect.

After the LZA enabled Security Hub I noticed that it is still using the Local configuration setting:

<img width="845" alt="Screen Shot 2024-04-11 at 4 43 17 PM" src="https://github.com/awslabs/landing-zone-accelerator-on-aws/assets/30264087/5c4ab998-484c-41a3-aee7-54abbf71d0a8">

If you enable Central configuration you can apply the Standards at the OU level. Can I configure the LZA to use this feature? would deploymentTargets then apply the Standards accordingly? 

- [ ] Version: 1.6.2
- [ ] Region: us-east-1
- [ ] Was the solution modified from the version published on this repository? No 
- [ ] If the answer to the previous question was yes, are the changes available on GitHub?
- [ ] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the services this solution uses? no limits
- [ ] Were there any errors in the CloudWatch Logs?

Thank you.

Juan.

**Comment by mongrol** [comment]
I don't believe Consolidated findings are a duplicate of this. This issue is about whether LZA allows the enabling (and subsequent usage) of Central Configuration of Security Hub. Consolidated Findings are a General Setting. In my testing clickopsing Central Configuration is reverted back to Local on the next run of LZA.

**Comment by fdjahnke** [comment]
Hello @jaristizabalc, thank you for filing this issue!

This has been addressed. Since **v1.15.0**, the `deploymentTargets` property on each Security Hub `standards[]` entry is honored — the standard is only applied to the accounts/OUs you list (for example, Staging), instead of everywhere. You're on **v1.6.2**, where that property had no effect, which matches what you observed. Upgrading to the latest release (**v1.15.5**) will resolve the OU-scoping behavior described here.

A separate note on your follow-up question: LZA configures Security Hub using Local configuration, not Security Hub Central Configuration, so OU-level targeting is achieved through `deploymentTargets` rather than central configuration policies. Native support for Security Hub Central Configuration is being tracked as a feature request in #505 — please follow and +1 that issue for updates.

Closing this issue as resolved. Please feel free to follow up here (ping me so I see it) if upgrading doesn't resolve the OU-scoping behavior.
